src/Gelf/Message.php

```diff
@@ -297,6 +297,15 @@ public function toArray()
         }
 
         // return after filtering false, null and empty strings
-        return array_filter($message, 'strlen');
+        return array_filter($message, array($this, 'filter'));
+    }
+
+    /**
```

Comment:
You may be right. I'll try to find out why the initial implementation was/is that way.